### ISSUE TITLE
Correct heading <h1>-<h6> order

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1614]
## Problem

The headings in the web pages often skip levels and go from `<h1>` to `<h3>` and `<h5>`. This is bad for accessibility and should be fixed.

## Description

The order of the headings on the page is important for screen reader users. It helps them navigate the page more effectively [^1].
We often use the headings for visual style instead of the page organization. Instead, we should use the correct order of headings `<h1> - <h6>`, but add the corresponding Tailwind styles to make the heading appearance match the design.

## Notes on implementation
The visual styles of the headings should match the [Figma design mockups](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release). Please note that Openverse uses Tailwind, and it will be necessary to add Tailwind styles to the headings to ensure that the font sizes match the designs. There is an [Openverse Tailwind preview](https://wordpress.github.io/openverse-frontend/tailwind/#Font%20Sizes) page that helps find the classes that match the needed styles.

The font sizes are usually different for mobile pages and for the pages above the mobile threshold. To implement that, you would need to use the base font size class without a prefix, and add a `md:` prefix to the font sizes used on pages wider than `md` breakpoint (tablet, desktop and up).

Since there are several pages, it is probably best to split this issue into several smaller ones:

- [x] #1615
- [x] #1616
- [x] #1698
- [x] #1617
- [x] #1618

### Notes
[^1]: More on how screen readers can use the headings and why the order is important: https://equalizedigital.com/accessibility-checker/incorrect-heading-order/

**Comment by RARNE12** [comment]
@obulat , @rarne12 and @kickeyhickey would like to continue working on these issues. @kickeyhickey and I worked on issue #1588, which is waiting for your final review. We can work on issue #1615-1617. We see we needed to style the headers, should we do our best to match with Tailwind CSS utility classes, or do you have any specific any style recommendations you want us to use? Thank you in a advance for any advice you can give us!

**Comment by obulat** [comment]
@RARNE12, thank you for your contributions!
You can start working on #1615 and #1616 (#1617 was assigned a couple of days ago).

After changing the heading level, you would need to update the styles to match the [Figma mock-ups](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=133%3A6869). Do note that the mobile styles are different from the desktop styles, so you would use two Tailwind style classes: one without a prefix for the mobile version, and one for the desktop (wider than `md` breakpoint) with `md:` prefix. To lookup the Tailwind classes you can use the [Openverse Tailwind Preview page](https://wordpress.github.io/openverse-frontend/tailwind/)

Don't hesitate to ask any questions you might have :)

**Comment by obulat** [comment]
@RARNE12, @kickeyhickey, @Jbudd512, @ramadanomar, thank you all so much for working on these issues! All of the linked issues have been resolved 🎉
